### DISCUSSION TITLE
Multiple instance of Artemis Operator inside the same k8s cluster

### DISCUSSION THREAD

**Discussion by lordigon** [arkmq-org/arkmq-org-broker-operator#808] [Q&A] [answered]
Is it possible to install within the same k8s cluster more Artemis operator instances? The goal would be to have different artemis operator/broker one for each namespace to simulate multiple installations we want to use for our tests.
Is it possible to expose in that case an acceptor for each artemis broker?
If it is possible any advice for that use case?

Best regards.

**Comment by brusdev** [comment, marked as answer]
> Is it possible to install within the same k8s cluster more Artemis operator instances? The goal would be to have different artemis operator/broker one for each namespace to simulate multiple installations we want to use for our tests.

Yes, it is possible to install within the same k8s cluster more Artemis operator instances if any instance watch only its namespace, see https://github.com/artemiscloud/activemq-artemis-operator/blob/main/docs/getting-started/quick-start.md#deploying-the-operator

> Is it possible to expose in that case an acceptor for each artemis broker? If it is possible any advice for that use case?

Yes, it is possible to expose in that case an acceptor for each artemis broker but you could get #803 if the you have multiple ActiveMQArtemis CRs with the same name
